dist/mobx-async-store.cjs.js

```diff
@@ -24,6 +24,7 @@ var cloneDeep = require('lodash/cloneDeep');
 var isEqual = require('lodash/isEqual');
 var isObject = require('lodash/isObject');
 var findLast = require('lodash/findLast');
+var lodash = require('lodash');
```

Comment:
is is possible to just import union instead of the entire lodash library similar to the imports above?